source/components/JsxParser.js

```diff
@@ -74,7 +74,8 @@ export default class JsxParser extends Component {
 
   parseElement = (element, key) => {
     const { bindings = {}, components = {} } = this.props
-    const { children = [], openingElement: { attributes, name: { name } } } = element
+    const { children = [], openingElement = {} } = element
+    const { attributes = [], name: { name = element.type } = {} } = openingElement
```

Comment:
Not sure I love defaulting this to `element.type`, but will redress if it causes errors.